theme/static/js/hs_file_browser.js

```diff
@@ -107,16 +109,31 @@ function updateSelectionMenuContext() {
     var flagDisableGetLink = false;
     var flagDisableCreateFolder = false;
 
+    var maxSize = MAX_FILE_SIZE * 1024 * 1024; // convert MB to Bytes
+
     if (selected.length > 1) {
         flagDisableRename = true; 
         flagDisableOpen = true;
         flagDisablePaste = true;
         flagDisableZip = true;
         flagDisableSetGeoRasterFileType = true;
         flagDisableGetLink = true;
+        
+        for (var i = 0; i < selected.length; i++) {
+            var size = parseInt($(selected[i]).find(".fb-file-size").attr("data-file-size"));
+            if (size > maxSize) {
+                flagDisableDownload = true;
+            }
+        }
+        $("#fb-download-help").toggleClass("hidden", !flagDisableDownload);
```

Comment:
Correct. This behavior is consistent with existing one. For example, if a folder is included in the selection the operation is disabled as well.